docs/docusaurus.config.js

```diff
@@ -0,0 +1,143 @@
+// @ts-check
+// `@type` JSDoc annotations allow editor autocompletion and type checking
+// (when paired with `@ts-check`).
+// There are various equivalent ways to declare your Docusaurus config.
+// See: https://docusaurus.io/docs/api/docusaurus-config
+
+import {themes as prismThemes} from 'prism-react-renderer';
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: 'Hoverkraft Theme',
+  tagline: 'Official Docusaurus theme for Hoverkraft projects',
+  favicon: 'img/favicon.ico',
+
+  // Set the production url of your site here
+  url: 'https://hoverkraft-tech.github.io',
+  // Set the /<baseUrl>/ pathname under which your site is served
+  // For GitHub pages deployment, it is often '/<projectName>/'
+  baseUrl: '/docusaurus-theme/',
+
+  // GitHub pages deployment config.
+  organizationName: 'hoverkraft-tech', // Usually your GitHub org/user name.
+  projectName: 'docusaurus-theme', // Usually your repo name.
+
+  onBrokenLinks: 'throw',
+  onBrokenMarkdownLinks: 'warn',
+
+  // Even if you don't use internationalization, you can use this field to set
+  // useful metadata like html lang. For example, if your site is Chinese, you
+  // may want to replace "en" with "zh-Hans".
+  i18n: {
+    defaultLocale: 'en',
+    locales: ['en'],
+  },
+
+  presets: [
+    [
+      'classic',
+      /** @type {import('@docusaurus/preset-classic').Options} */
+      ({
+        docs: {
+          path: 'content',
+          sidebarPath: './sidebars.js',
+          editUrl:
+            'https://github.com/hoverkraft-tech/docusaurus-theme/tree/main/docs/',
+        },
+        blog: {
+          showReadingTime: true,
+          feedOptions: {
+            type: ['rss', 'atom'],
+            xslt: true,
+          },
+          editUrl:
+            'https://github.com/hoverkraft-tech/docusaurus-theme/tree/main/docs/',
+        },
+        theme: {
+          customCss: './src/css/custom.css',
+        },
+      }),
+    ],
+  ],
+
+  // Use the Hoverkraft theme
+  themes: ['@hoverkraft/docusaurus-theme'],
```

Comment:
Is it possible to use the "local" theme with relative path or something like that